data/formats-data.js

```diff
@@ -3339,7 +3346,7 @@ exports.BattleFormatsData = {
 		tier: "NU",
 	},
 	spheal: {
-		randomBattleMoves: ["substitute", "protect", "toxic", "surf", "icebeam", "yawn", "superfang"],
```

Comment:
yawn should be kept to phaze

Reply:
but does it really deserve a moveslot?

Reply:
i would believe so.  it applies pressure on the opponent

Reply:
I mean I know but idk when I'd want it over one of the other four